pkg/tsdbctl/tsdbctl.go

```diff
@@ -57,23 +58,34 @@ func NewRootCommandeer() *RootCommandeer {
 	commandeer := &RootCommandeer{}
 
 	cmd := &cobra.Command{
-		Use:          "tsdbctl [command]",
-		Short:        "V3IO TSDB command-line interface",
+		Use:          "tsdbctl [command] [arguments] [flags]",
+		Short:        "V3IO TSDB command-line interface (CLI)",
 		SilenceUsage: true,
 	}
 
 	defaultV3ioServer := os.Getenv("V3IO_SERVICE_URL")
 
-	cmd.PersistentFlags().StringVarP(&commandeer.verbose, "verbose", "v", "", "Verbose output")
-	cmd.PersistentFlags().Lookup("verbose").NoOptDefVal = "debug"
-	cmd.PersistentFlags().StringVarP(&commandeer.dbPath, "table-path", "t", "", "sub path for the TSDB, inside the container")
-	cmd.PersistentFlags().StringVarP(&commandeer.v3ioPath, "server", "s", defaultV3ioServer, "V3IO Service URL - ip:port")
-	cmd.PersistentFlags().StringVarP(&commandeer.cfgFilePath, "config", "g", "", "path to yaml config file")
-	cmd.PersistentFlags().StringVarP(&commandeer.container, "container", "c", "", "container to use")
-	cmd.PersistentFlags().StringVarP(&commandeer.username, "username", "u", "", "user name")
-	cmd.PersistentFlags().StringVarP(&commandeer.password, "password", "p", "", "password")
-
-	// add children
+	cmd.PersistentFlags().StringVarP(&commandeer.verbose,
+		"verbose", "v", "", "Verbose output")
+	cmd.PersistentFlags().Lookup("verbose").NoOptDefVal = defaultVerbosityLevel
+	cmd.PersistentFlags().StringVarP(&commandeer.dbPath, "table-path", "t", "",
+		"[Required] Path to the TSDB table within the configured\ndata container. Examples: \"mytsdb\"; \"/my_tsdbs/tsdbd1\".")
+	// We don't enforce this flag (cmd.MarkFlagRequired("table-path")),
+	// although it's documented as Required, because this flag isn't required
+	// for the hidden `time` command + during internal tests we might want to
+	// configure the table path in a configuration file.
+	cmd.PersistentFlags().StringVarP(&commandeer.v3ioPath, "server", "s", defaultV3ioServer,
+		"Web-gateway (web-APIs) service endpoint of an instance of\nthe Iguazio Continuous Data Platfrom, of the format\n\"<IP address>:<port number=8081>\". Examples: \"localhost:8081\"\n(when running on the target platform); \"192.168.1.100:8081\".")
+	cmd.PersistentFlags().StringVarP(&commandeer.cfgFilePath, "config", "g", "",
+		"Path to a YAML TSDB configuration file. When this flag isn't\nset, the CLI checks for a "+config.DefaultConfigurationFileName+" configuration file in the\ncurrent directory. CLI flags override file confiugrations.\nExample: \"~/cfg/my_v3io_tsdb_cfg.yaml\".")
```

Comment:
I think word `YAML` is useless in this sentence

Reply:
I think it's helpful because it clarifies in which format we expect the configuration file to be.